config/build-and-release.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and release Docker image
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  Deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          body: |
+            Changes in this Release
+            - ${{ github.ref }}
+          draft: false
+          prerelease: false
+      - name: Build and Push Demo Docker Image
+        uses: docker/build-push-action@v2
+        with:
+          push: true # Pushes after image build
+          context: ./
+          tags: |
+            ghcr.io/${{ github.repository }}:${{ github.ref_name }}
```

Comment:
Does this mean no longer using/having the latest tag?

Reply:
This has been updated: https://github.com/lightstep/telemetry-generator/pull/24/files#diff-1bb533fec61100e3fdb5975dbaf121ca33d722ae03aa622aa67c41272a73e274R39

now means that the Docker image will have both the newest tag && `latest`. See how it works on my personal fork now: https://github.com/tratnayake/telemetry-generator/pkgs/container/telemetry-generator